README.md

```diff
@@ -3,14 +3,18 @@
 This guide is to help you setup the following required software during the Black Codher bootcamp.
 
 1. [Windows Terminal](#windows-terminal) **THIS IS FOR WINDOWS USERS ONLY**
+1. [Setup Home Path Alias](#setup-home-path-alias-in-windows-terminal) **THIS IS FOR WINDOWS USERS ONLY**
 1. [Git](#git)
+   - [Setup SSH Keys for GitLab](#setup-ssh-keys-for-gitlab)

```

Comment:
GitLab?

Reply:
My bad! I've been using GitLab for work for the past couple of months. It's taken over my life. I'll correct it now.